its/src/org/sonarlint/eclipse/its/StandaloneAnalysisTest.java

```diff
@@ -74,6 +76,56 @@ public class StandaloneAnalysisTest extends AbstractSonarLintTest {
   public void removeEnhancedConnectedModeInformationPerTest() {
     System.setProperty("sonarlint.internal.ignoreEnhancedFeature", "true");
     System.setProperty("sonarlint.internal.ignoreMissingFeature", "true");
+    System.setProperty("sonarlint.internal.ignoreNoAutomaticBuildWarning", "true");
+  }
+  
+  @After
+  public void enableAutomaticWorkspaceBuild() {
+    if ("oldest".equals(System.getProperty("target.platform"))) {
+      var preferences = GeneralWorkspaceBuildPreferences.open();
+      preferences.enableAutomaticBuild();
+      preferences.ok();
+    }
+  }
+  
+  @Test
+  public void analyze_automatic_workspace_build_disabled() {
+    // INFO: We only want to run it on one axis and the "oldest" ITs take the shortest!
```

Comment:
On the long run, it might be cleaner to use JUnit categories to partition tests.

Reply:
I haven't worked with them yet, but it is a valid point!